config/jobs/kubernetes/sig-node/local-e2e-containerized.yaml

```diff
@@ -64,3 +71,10 @@ periodics:
       # docker-in-docker needs privileged mode
       securityContext:
         privileged: true
+      resources:
+        requests:
+          # these are both a bit below peak usage during build
+          # this is mostly for building kubernetes
+          memory: "9000Mi"
```

Comment:
running this on every PR for a long time might (?) be an issue. kind is only running a couple of CI jobs for 1hr plus, and some < 20m presubmits on a small repo (kind).

Reply:
sorry this dropped off my radar, I'm not really sure how to fix this issue.

Reply:
I don't think we have support for this.